trillian.proto

```diff
@@ -139,6 +139,10 @@ message Tree {
   // The public key used for verifying tree heads and entry timestamps.
   // Readonly.
   PublicKey public_key = 14;
+
+  // Interval after which a new signed root is produced even if there have been
+  // no submission.  If zero, this behavior is disabled.
+  int64 force_new_root_millis = 15;
```

Comment:
On a quick look, the protobuf `Duration` type doesn't look helpful -- it has separate [seconds and nanoseconds](https://github.com/google/protobuf/blob/master/src/google/protobuf/duration.proto#L103-L117) and so would need non-trivial conversion functions.